EIPS/eip-1679.md

```diff
@@ -24,52 +24,51 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 
 ### Accepted EIPs
 
-- TBD
+- [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
+- [EIP-2024](https://github.com/ethereum/EIPs/pull/2129): Blake2b Precompile
 
```

Comment:
I think there was an agreement on 1108.

Reply:
I remembered something on it, but I wasn't totally sure. I can add that one in

Reply:
From meeting 62:
> Hudson Jameson: EIP-1108 is approved in principle, pending a benchmark which will be reviewed at the next meeting.